src/pages/user/order/OrderWait.tsx

```diff
@@ -0,0 +1,140 @@
+import HorizontalDivider from '@components/common/divider/HorizontalDivider';
+import { Color } from '@resources/colors';
+import { useEffect } from 'react';
+import useAdminOrder from '@hooks/admin/useAdminOrder';
+import useBlockPopState from '@hooks/useBlockPopState';
+
+const Container = styled.div`
+  width: 100%;
+  height: 100%;
+  padding-top: 50px;
+`;
+
+const SubContainer = styled.div`
+  box-sizing: border-box;
+  width: 100%;
+  padding: 20px 20px 0 20px;
+  ${colFlex({ justify: 'center', align: 'center' })}
+`;
+
+const ContentsContainer = styled.div`
+  box-sizing: border-box;
+  width: 100%;
+  padding: 12px 10px;
+  gap: 20px;
+  border-radius: 9px;
+  border: 0.5px solid #939393;
+  ${colFlex({ justify: 'center', align: 'start' })}
+`;
+
+const OrderInfoContainer = styled.div`
+  box-sizing: border-box;
+  width: 100%;
+  padding: 0 10px;
+  ${rowFlex({ justify: 'space-between', align: 'center' })};
+`;
+
+const StyledLabel = styled.div`
+  font-size: 13px;
+  font-weight: 600;
+`;
+
+const DescriptionContainer = styled.div`
+  width: 100%;
+`;
+
+const DescriptionText = styled.div`
+  box-sizing: border-box;
+  border-radius: 10px;
+  width: 100%;
+  font-size: 13px;
+  font-weight: 500;
+  color: #898989;
+  text-align: center;
+  box-sizing: border-box;
+  background: ${Color.LIGHT_GREY};
+  padding: 20px 30px;
+  word-break: keep-all;
+  ${colFlex({ justify: 'center', align: 'center' })}
+`;
+
+function OrderWait() {
+  const navigate = useNavigate();
+  const [searchParams] = useSearchParams();
+  const workspaceId = searchParams.get('workspaceId');
+  const tableNo = searchParams.get('tableNo');
+  const orderId = searchParams.get('orderId');
+  const isTossAvailable = searchParams.get('isTossAvailable') === 'true';
+
+  const workspace = useAtomValue(userWorkspaceAtom);
+  const adminOrders = useAtomValue(adminOrdersAtom);
+
+  const { fetchTodayOrders } = useAdminOrder(workspaceId || undefined);
+
+  useBlockPopState();
+
+  useEffect(() => {
+    const intervalId = setInterval(() => {
+      fetchTodayOrders();
+    }, 1000);
+
+    return () => clearInterval(intervalId);
+  }, [workspaceId]);
+
+  const currentOrder = adminOrders.find((order) => order.id === Number(orderId));
+  const currentOrderStatus = currentOrder?.status;
+  const totalPrice = currentOrder?.totalPrice ?? 0;
+
+  const navigateToComplete = () => {
+    navigate(`/order-complete?orderId=${orderId}&workspaceId=${workspaceId}&tableNo=${tableNo}`);
+  };
+
+  useEffect(() => {
+    if (currentOrderStatus === 'PAID') {
+      navigateToComplete();
+    }
+  }, [currentOrderStatus]);
+
+  const waitingText = isTossAvailable
+    ? '현재 결제 확인 중입니다. 토스 앱이 자동으로 열리지 않았다면, 위 계좌로 직접 송금해주세요. 결제가 확인되면 자동으로 주문 내역 페이지로 이동합니다.'
+    : '위 계좌로 송금을 완료해주세요. 송금 후에는 잠시만 기다려주세요. 결제가 확인되면 자동으로 주문 내역 페이지로 이동합니다.';
+
+  const orderButtonText = currentOrderStatus === 'NOT_PAID' ? '결제 확인 중 . . .' : '주문 확인 페이지로 이동';
```

Comment:
```suggestion
  const orderButtonText = '결제 확인 중 . . .';
```

그리고 디자인상 결제 확인중의 ... 도 확인중인 것 처럼 늘어났다가 줄었다가 해야될 것 같습니다.
그리고 interval로 확인하고 있는 이상 여기에서 조건적으로 보여줄 때는 없을 것 같습니다.

Reply:
그렇다면 해당 애니메이션이 추가된 버튼은 기존에 사용하던 공용 컴포넌트가 아니라 OrderWait 컴포넌트 안에 새로 선언하면 될까요?

Reply:
음 그냥 Text를 state로 선언해도 괜찮지 않을까요?
따로 해야할 것 같다면 따로하셔도 됩니다~